app/src/main/java/io/homeassistant/companion/android/bluetooth/ble/TransmitterManager.kt

```diff
@@ -0,0 +1,84 @@
+package io.homeassistant.companion.android.bluetooth.ble
+
+import android.bluetooth.BluetoothAdapter
+import android.bluetooth.le.AdvertiseCallback
+import android.bluetooth.le.AdvertiseSettings
+import android.content.Context
+import org.altbeacon.beacon.Beacon
+import org.altbeacon.beacon.BeaconParser
+import org.altbeacon.beacon.BeaconTransmitter
+object TransmitterManager {
```

Comment:
Should this be injected like the `ThemeManager`?

Reply:
Not sure if this is required - let me know.